app/models/manual.rb

```diff
@@ -0,0 +1,173 @@
+class Manual
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  attr_accessor :content_id, :base_path, :title, :summary, :body, :updated_at, :public_updated_at, :publication_state, :update_type
```

Comment:
As you have explicit getters/setters for `updated_at` and `public_updated_at` below, they should probably be removed from here to avoid confusion